qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -26,14 +26,18 @@
             blk_num = 1
             repeat_times = 3
         - multi_pci:
-            blk_num = 2
+            blk_num = 3
             repeat_times = 3
-            images += " stg1"
+            images += " stg1 stg2"
             boot_drive_stg1 = no
+            boot_drive_stg2 = no
```

Comment:
Put these params into .py as optional params might be a good idea.

Reply:
Hi hachen, 
it's a param based to the case multi_pci, if test with_plug, boot_drive_stg* = no, then hotplug stg0, stg1, stg2; if test without_plug, boot_drive_stg*=yes, it means will add disk at booting the guest up, then do hotunplug stg0, stg1, stg2. What do you mean that "put these params into .py as optional params"? if added to .py, if also need to add some condition to differ cases for one_pci and cases for multi_pci, if yes, if it's more  complex these case?

Thanks~

Reply:
OK,Let's keep it this way.